### PR TITLE
Filter Unicode posts from Tumblr

### PR DESCRIPTION
**Pull request by eatnumber1**

### CODE REVIEW

**Comment by eatnumber1** [comment]
Note that this change merge conflicts with the equivalent change I made to Chase's repository. Y'all should figure out which one is the canonical one.

https://github.com/chaseberry/CSH-Infosys/pull/1

File: tumblr.py [hunk 1]
```diff
@@ -51,4 +60,8 @@ def postToSign(text, FILE_ID):
         creds['oauthSecret']
     )
 
-    postToSign(getPostWithLimit(client, args.user), fileID)
+    postToSign(
+		getPostWithoutUnicode(
+			lambda: getPostWithLimit(
+				lambda: getPost(client, args.user))),
+		fileID)
```
Comment (harlanhaskins) [inline tumblr.py]: Dem doubly-nested lambdas

Comment (eatnumber1) [inline tumblr.py]: I would have used list comprehensions if Python was lazy about them :(

**Comment by eatnumber1** [comment]
Bump

**Pull request merged**